filebeat/scripts/module/docs.asciidoc

```diff
@@ -1,3 +1,30 @@
+:modulename: {module}
+
 == {module} module
 
 This is the {module} module.
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+TODO: document with what versions of the software is this tested
+
+
+include::../include/running-modules.asciidoc[]
+
+[float]
+=== Example dashboard
+
+This module comes with a sample dashboard. For example:
+
+TODO: include an image of a sample dashboard
+
+include::../include/configuring-intro.asciidoc[]
+
+TODO: provide an example configuration
+
+include::../include/config-option-intro.asciidoc[]
+
+TODO: document: fileset variables
```

Comment:
Should we expand the instructions here and provide an example? Or will that confuse users? If you think this would be confusing to module developers, we can provide feedback when the module is submitted for review.

````
TODO: document: fileset variables. If you're describing a variable that's common to other modules, you
can reuse shared descriptions by including the relevant file. For example:
[float]
==== `access` log fileset settings

include::../include/var-paths.asciidoc[]